### ISSUE TITLE
Unable to persist data on sqlite DB (fresh new setup)

### ISSUE THREAD

**Issue by AliChegini** [vapor/vapor#2102]
I made a fresh new setup today and tried to save a new todo. I used postman to make a post request with id and title in the body of the request. Entry that I post reaches the server as my print statement in create(_ req: Request) get executed and I see 200 OK status in postman for my request, but the new entry does not get saved on sqlite db file! When I make a get request to show all todos, it returns an empty array.

I modified Configure.swift file so instead of saving in memory I saved the DB file in my desktop folder. I opened the DB file using DB viewer. Surprisingly posted data was not saved in the DB file. (Even without this modification issue exist using in memory storage)
I added a manual entry with DB viewer, saved the file, made a new get request and i could see the manually added data returned correctly. 
Simply put, I can't persist a new todo using post request. How can I solve this issue?

Please note I just followed the instruction in https://docs.vapor.codes/3.0/install/macos/ to setup a new project and there is no modification other than what I explained.

<img width="570" alt="Screenshot 2019-11-21 at 17 00 29" src="https://user-images.githubusercontent.com/14887288/69354603-aca40900-0c80-11ea-9afe-c5d5a714d6f1.png">

* Vapor Framework version: unable to determine (see the reason in screenshot)
* Vapor Toolbox version: 3.1.10
* OS version: 10.15.1

Repo: https://github.com/AliChegini/Swift-Server-Test

**Comment by grundoon** [comment]
If you are supplying the ID yourself, you should use `todo.create(on:)` instead of `todo.save(on:)`. (Most of the time you will want to use `save(on:)` and have the ID auto-generated.)

> Saves the model, calling either create(...) or update(...) depending on whether the model already has an ID.
> 
> If you need to create a model with a pre-existing ID, call create instead. 

https://api.vapor.codes/fluent/3.2.0/Fluent/Protocols/Model.html#/s:6Fluent5ModelPAAE4save2on3NIO15EventLoopFutureCyxG11DatabaseKit0I11Connectable_p_tF

**Comment by AliChegini** [comment]
@grundoon Thanks for your quick response. It's my first experiment with Vapor and I was just trying to supply dummy data in order to see if it works and I did not pay attention to ID sensitivity. It makes more sense now.
